### PR TITLE
Remove overlapping variables from table of content.

### PR DESCRIPTION
**Pull request by Dhruvi16**
## Description

Removes overlapping variables in Table of Content component. The same variables with same value are declared in Sidebar menu so repeating them in TAble of content was unnecessary.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/stories/table_of_contents.stories.mdx [hunk 1]
```diff
@@ -25,15 +25,15 @@ The component has two states, active and inactive. The hover state of inactive e
       <aside class="menu toc">
         <ul class="menu-list">
           <li>
-            <a class="is-active link">Section one, active state</a>
+            <a class="is-active">Section one, active state</a>
             <ul>
-              <li><a class="inactive-link"><i class="icon circle-filled"></i>Child section</a></li>
```
Comment (hugosolar) [inline src/stories/table_of_contents.stories.mdx:30]: This `.inactive-link` class was removed because there isn't an "inactive" state of links right?

Comment (Dhruvi16) [reply src/stories/table_of_contents.stories.mdx:30]: Yes. It was added by me earlier but it was not required.

**Review by hugosolar — APPROVED** [review]

File: src/styles/table_of_contents.scss [hunk 2]
```diff
@@ -23,11 +15,6 @@ $menu-list-link-padding: 0.5rem 0;
     text-decoration: none;
   }
 
-  .menu-list .inactive-link {
-    @extend .link;
-    color: $color-tomato;
-  }
-
   i {
```
Comment (dhruvkb) [inline src/styles/table_of_contents.scss]: If this is for the `i.icon` element, use the class `.icon` here.

File: src/styles/table_of_contents.scss [hunk 3]
```diff
@@ -1,11 +1,3 @@
...
-$menu-item-active-background-color: none;
-$menu-item-active-color: $color-dark-slate-gray;
-$menu-list-border-left: none;
-$menu-nested-list-margin: 0;
-$menu-nested-list-padding-left: 0;
-$menu-list-link-padding: 0.5rem 0;
-
 @import "bulma/sass/components/menu.sass";
 
 .toc {
```
Comment (dhruvkb) [inline src/styles/table_of_contents.scss]: For consistency with `.table-of-progress`, rename this to `.table-of-contents`.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
This review also applies to `.side-navigation` in `sidebar_menu.scss`.

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**